packages/flutter/lib/src/painting/shader_warm_up.dart

```diff
@@ -157,22 +155,18 @@ class DefaultShaderWarmUp extends ShaderWarmUp {
       canvas.save();
       for (ui.Paint paint in paints) {
         canvas.drawPath(paths[i], paint);
-        canvas.translate(80.0, 0.0);
```

Comment:
One reason that we added these 80.0 offsets is to visualize the warm up draws in `examples/layers/raw/shader_warm_up.dart`. Can you please add a flag here to preserve the offset in that debug app? (Maybe it's nice to do some refactor to force each draw within an 100x100 area, and automatically stack them together for actual warm-up and spread them for debugging.)

Reply:
Done

Reply:
Can you please also set the `_drawCallSpacing` to 80.0 in `examples/layers/raw/shader_warm_up.dart`?

Reply:
@gaaclarke : any comment on this?

Reply:
Done.